ios/FluentUI/Navigation/NavigationBar.swift

```diff
@@ -156,6 +156,19 @@ open class NavigationBar: UINavigationBar {
         return nil
     }
 
+    @objc public func barButtonItemView(tag: Int) -> UIView? {
+        let leftBarButtonItemViews = leftBarButtonItemsStackView.arrangedSubviews
+        let rightBarButtonItemViews = rightBarButtonItemsStackView.arrangedSubviews
+        let totalBarButtonItemViews = leftBarButtonItemViews + rightBarButtonItemViews
```

Comment:
in regular title mode the tooltip seems be bit off? 
![Screen Shot 2021-08-18 at 2 01 37 PM](https://user-images.githubusercontent.com/20715435/129971872-7f7d101f-370d-4e52-b890-e3de3479a5fb.png)

Reply:
Yes, this is a bit off and the reason is the navbar's height which depends on the regular or large title mode so IMO whosoever creating the tooltip should take care of this. In the demo controller, I took care of this by checking for the large title thing and then setting `y` offset as 5 for the tooltip.

There is one more thing we can do is expose an API that will tell the difference between the height of the navbar in large and regular title mode. `Constants.expandedContentHeight - Constants.normalContentHeight`

But, it's not necessary that users of our API(`barButtonItemView(with tag:)`) necessarily want offset equal to the difference between the height of the navbar in regular and large title mode like in our case the difference is `4` based on current constants value but I set offset as `5` in the demo controller in case of regular title mode because that is looking even better. (Check the below screenshots where the offset is `4` in the first screenshot and `5` in second screenshot)
<img width="340" alt="Screenshot 2021-08-19 at 11 21 27 AM" src="https://user-images.githubusercontent.com/33866787/130016369-621ca020-54e8-47df-b7d4-216e8f187ca8.png">

Reply:
no I don't think that analysis is correct. I think the right and left stack view have their own stack view with the custom uibutton (just like the large title case) but it isn't actually rendered on the screen when it is "system" style navigation bar. the uivew that user actually see is the real uinavigationbar baritembutton provided by iOS.

Reply:
@harrieshin I am not sure where my understanding is wrong because here's what my analysis was based on:
1. `updateViewsForLargeTitlePresentation` method in NavigationBar.swift is showing or hiding the content stack view which contains the left and right bar button items stack views.
2. In the demo controller we have the option to have the regular title for both "system" and "primary" navigation bar styles. <img width="326" alt="Screenshot 2021-08-20 at 1 21 22 PM" src="https://user-images.githubusercontent.com/33866787/130199531-e2669dac-85c1-42e2-b947-8c3ec10e86e1.png">

Now, if I write code based on title condition:`let offset: CGPoint = navigationItem.usesLargeTitle ? .init(x: 0, y: 0) : .init(x: 0, y: -4)` then it works fine like shown in the screenshots.
| <img width="332" alt="Screenshot 2021-08-20 at 1 25 14 PM" src="https://user-images.githubusercontent.com/33866787/130200725-c743c55f-edfe-493f-8595-43382432c341.png"> | <img width="332" alt="Screenshot 2021-08-20 at 1 25 30 PM" src="https://user-images.githubusercontent.com/33866787/130200736-e2380e38-c578-4e10-8246-e66baabb5aa8.png"> |

Reply:
The point I am trying to convey is that the UIView you are returning for regular title with tag="1" you are actually returning incorrect view. (in the demo app if you say view.background = .yellow for example, you will see it will be no op)the rect happens to be somewhat close so your demo code of doing -4 works for tooltip but if this public function is trying to access the actual view then they are not getting the right UIView to interact with it. so I think the logic of retrieving  the right view needs to be different for showsLargeTitle case.

Reply:
My bad!! I should have made the API to handle both the scenarios(large title or regular title). Can you please check now?